pkg/doc/verifiable/presentation.go

```diff
@@ -0,0 +1,283 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package verifiable
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+const basePresentationSchema = `
```

Comment:
Where is this schema coming from?

Reply:
This schema is built by us according to the Verifiable Presentation data model definition (mainly based on [this section](https://www.w3.org/TR/vc-data-model/#presentations-0)).